include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -15,6 +15,7 @@
 #endif
 
 #include <string>
+#include <vector>
```

Comment:
The error on Windows is along the lines of 'disambiguation between x and y not possible'
On Unix there is no error, just a crash trying to get the address when print a std::vector\<bool\>

I believe Windows will also crash because the include failure (a separate issue).